RecoTracker/LSTCore/src/alpaka/MiniDoublet.h

```diff
@@ -1030,11 +1024,11 @@ namespace SDL {
 
         if (module_eta < 0.75)
```

Comment:
for a followup: these should be changed to floats to avoid up-conversion to doubles
```suggestion
        if (module_eta < 0.75f)
```